index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+      integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h5>Ismail Changezi</h5>
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+      <section>
+        <h4>Hey There.<br />I'm Ismail</h4>
+        <h6>I'm a Software Developer</h6>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="icons">
+          <ul>
+            <li><a href="#"    <i class="fa-brands fa-github"></i></a></li>
+       <li><a href="#"<li><a href="#"<li><a href="#"<li><a href="#"   <i class="fa-brands fa-linkedin-in"></i></a></li>>
+       <li><a href="#"<li><a href="#"<li><a href="#"   <i class="fa-brands fa-angellist"></i></a></li>
+       <li><a href="#"<li><a href="#"   <i class="fa-brands fa-facebook"></i></a></li>
+        <li><a href="#"  <i class="fa-brands fa-twitter"></i></a></li>
+          </ul>
```

Comment:
- [ ] You mare need to style your work again to befit your current HTML structure. After you introduced the `a` tags as requested by the previous reviewer, you probably forgot to update your styles.
👇  Please screenshot below: 

## **Your social media icon design:** After adding.
![image](https://user-images.githubusercontent.com/84156714/185716504-f9a22865-bc2d-4d17-8909-45800d86d68e.png)

## Before 
![image](https://user-images.githubusercontent.com/84156714/185716666-6629fee1-b4a9-454c-8edb-25151a30db94.png)